src/license_text_normalizer/main.py

```diff
@@ -147,7 +148,8 @@ def _strip_words(line: str, words_to_strip: List[str]) -> str:
     return normalized_line.lstrip()
 
 
-def _is_repeated_non_alnum(line: str) -> bool:
-    if not len(line) > 1:
+def _is_line_non_alnum(line: str, delimiters: List[str]) -> bool:
+    if not (len(line) > 1 and line[0] in delimiters):
```

Comment:
I think we can broaden this to not limit ourselves to just the currently configured delimiter characters. I think it's safe to say that if a line consists of just non-alphanumeric characters in a license text, then it should be safe to strip it out without affecting the semantics.